db/migrations/20201028151313_initial.js

```diff
@@ -0,0 +1,20 @@
+
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('card_games', function (table) {
+      table.increments('id').primary();
+      table.string('name');
+      table.string('instructions', 10000);
+      table.string('description', 10000);
```

Comment:
I added a similar comment to one of BD's PR and I think we should try to use the `table.text` if possible when dealing with large strings. I have not use them myself yet but I think it could be a little bit more semantic.

Reply:
I agree! I will change this for the `instructions` and `description`.